layouts/plugins/system/webauthn/manage.php

```diff
@@ -57,8 +57,14 @@
 ];
 extract(array_merge($defaultDisplayData, $displayData));
 
+if ($displayData['allow_add'] === false)
+{
+	$error = Text::_('PLG_SYSTEM_WEBAUTHN_CANNOT_ADD_FOR_A_USER');
+	$allow_add = false;
+}
+
 // Ensure the GMP or BCmath extension is loaded in PHP - as this is required by third party library
-if ((function_exists('gmp_intval') === false) && (function_exists('bccomp') === false))
+if (!$error && (function_exists('gmp_intval') === false) && (function_exists('bccomp') === false))
```

Comment:
```suggestion
if ($allow_add && function_exists('gmp_intval') === false && function_exists('bccomp') === false)
```
I understand the result will be the same but I find it easier to read without the `!`. Plus it will be a consistent checking `$allow_add` as below.